### PR TITLE
Missing docs and tweaks

### PR DESCRIPTION
**Pull request by einarf**
* `arcade.resources` was not documented
* Update resource handle guide to refer to "handles" instead of "handlers" and link to api docs
* Delete guide about edge artifacts because this more or less a non-existing problem now

### CODE REVIEW

**Review by pushfoo — APPROVED** [review]
Agreed: atlas changes make the old doc less relevant.

File: doc/programming_guide/resource_handlers.rst [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-Adding Your Own Resource Handlers
-=================================
+Adding Your Own Resource Handles
+================================
 
-Arcade provides a convenient way to locate asset through it's resource handlers. Arcade already has a number of
-Built-In sprites, images and other resources available for use inside the ``:resources:`` handler, which you
-may already be familiar with:
+Arcade provides a convenient way to locate asset through it's resource handles. Arcade already has a number of
+Built-In sprites, images and other resources available for use inside the ``:assert::``, and ``:system:``
```
Comment (pushfoo) [inline doc/programming_guide/resource_handlers.rst:7]: Typo:
```suggestion
Built-In sprites, images and other resources available for use inside the ``:assets:``, and ``:system:``
```

**Review by pushfoo — CHANGES_REQUESTED** [review]
On a second read-through, I spotted a typo.

File: doc/programming_guide/resource_handlers.rst (hunk 1 above)
Comment (einarf) [reply doc/programming_guide/resource_handlers.rst:7]: I'll get it in the resources vs assets pass.

**Pull request merged**